libs/datasets/combined_datasets.py

```diff
@@ -158,6 +158,9 @@ def datasource_regions(
 # so this region mask might not always be in use.
 NE_COUNTIES = RegionMask(AggregationLevel.COUNTY, states=["NE"])
 
+# 02/01/2022: CDC NC data is inaccurate due to an issue with categorizing
+# booster shots. So we instead use state-provided data
+NC_STATE = RegionMask(AggregationLevel.STATE, states=["NC"])
```

Comment:
I am not 100% sure but I think this could just be `Region.from_state("NC")`

Reply:
Good call, yeah it looks like it can just be the region. Just to make sure I'll create a snapshot with this (I haven't created one with the other changes yet, either). If it works okay I'll go ahead and merge